### PR TITLE
Project source code button added on home page

### LINKED ISSUE #8: Add project GitHub link button to home page
**Expected Behaviour**

Below the `Register Here` button, add a new button for project source code link and link it to https://github.com/jogendra/HealthLedger

**Would you like to work on the issue?**

Anyone can claim the issue :)

### PR DESCRIPTION
**Pull request by LalitSharma7**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [ ] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](url)

- Fixes #8 

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by Rattlecode17sh** [comment]
screenshot 50
Here is the screenshot. Link is not openining so i am attaching the screenshot

File: index.html [hunk 1]
```diff
@@ -63,6 +63,9 @@ <h3>Application for tracking Organs donations in hospitals and minimizing the sc
           <a href="Front end/register.html">
               <button class="btn btn-primary btn-simple btn-round btn-lg" type="button">Register here</button>
           </a>
+          <a href="https://github.com/jogendra/HealthLedger">
+              <button class="btn btn-primary btn-simple btn-round btn-lg" type="button">Project Source Code</button>
+          </a>
```
Comment (jogendra) [inline index.html]: Please put this button below the `And join our community` line and make it different from other buttons(Background and color wise, if possible add GitHub icon).

**Review by jogendra — CHANGES_REQUESTED** [review]

**Comment by Rattlecode17sh** [comment]
okay

**Comment by Rattlecode17sh** [comment]
live link

https://rattlecode17sh.github.io/HealthLedger/

**Comment by Rattlecode17sh** [comment]
Have a look and tell me if more changes are required

**Comment by jogendra** [comment]
@Rattlecode17sh Put a title text on button along with icon too : **Project Source Code**

**Comment by LalitSharma7** [comment]
screenshot 51
Have a look @jogendra

**Comment by LalitSharma7** [comment]
https://rattlecode17sh.github.io/HealthLedger/

**Review by jogendra — APPROVED** [review]
GTG!

**Pull request merged**